<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Surajkumars-MacBook-Pro.local" failures="16" tests="21" name="Command line test" time="0.448" errors="0" timestamp="2023-01-15T01:18:39 IST">
  <testcase classname="com.proj.base.MobileSetUp" name="@BeforeSuite setUp" time="0.787">
    <failure type="org.openqa.selenium.WebDriverException" message="Connection refused (Connection refused)
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:17:03&amp;apos;
System info: host: &amp;apos;Surajkumars-MacBook-Pro.local&amp;apos;, ip: &amp;apos;fe80:0:0:0:486:29c1:8c2c:88e3%en0&amp;apos;, os.name: &amp;apos;Mac OS X&amp;apos;, os.arch: &amp;apos;x86_64&amp;apos;, os.version: &amp;apos;13.1&amp;apos;, java.version: &amp;apos;11.0.15&amp;apos;
Driver info: driver.version: Driver">
      <![CDATA[org.openqa.selenium.WebDriverException: Connection refused (Connection refused)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'Surajkumars-MacBook-Pro.local', ip: 'fe80:0:0:0:486:29c1:8c2c:88e3%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '13.1', java.version: '11.0.15'
Driver info: driver.version: Driver
at io.appium.java_client.remote.AppiumCommandExecutor.lambda$6(AppiumCommandExecutor.java:262)
at java.base/java.util.Optional.orElseGet(Optional.java:369)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:261)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:363)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at io.appium.java_client.DefaultGenericMobileDriver.<init>(DefaultGenericMobileDriver.java:37)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:89)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:99)
at com.proj.driver.manager.mobile.local.AndroidManager.getDriver(AndroidManager.java:26)
at com.proj.driver.factory.mobile.local.LocalMobileDriverFactory.getDriver(LocalMobileDriverFactory.java:25)
at com.proj.driver.impl.mobile.LocalMobileDriverImpl.getDriver(LocalMobileDriverImpl.java:12)
at com.proj.driver.impl.mobile.LocalMobileDriverImpl.getDriver(LocalMobileDriverImpl.java:8)
at com.proj.driver.Driver.initDriverForMobile(Driver.java:60)
at com.proj.base.MobileSetUp.setUp(MobileSetUp.java:31)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
at java.base/java.net.PlainSocketImpl.socketConnect(Native Method)
at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:412)
at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:255)
at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:237)
at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
at java.base/java.net.Socket.connect(Socket.java:608)
at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:245)
at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:165)
at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
at okhttp3.RealCall.execute(RealCall.java:77)
at org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:103)
at io.appium.java_client.remote.AppiumCommandExecutor.lambda$1(AppiumCommandExecutor.java:162)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:105)
at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:193)
at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:225)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:250)
... 44 more
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeSuite setUp -->
  <testcase classname="com.proj.base.WebSetUp" name="@BeforeClass setUp" time="-1.673725713269E9">
    <skipped/>
  </testcase> <!-- @BeforeClass setUp -->
  <testcase classname="com.proj.base.WebSetUp" name="@AfterClass tearDown" time="-1.673725713313E9">
    <skipped/>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase classname="com.proj.base.WebSetUp" name="@BeforeClass setUp" time="-1.673725713339E9">
    <skipped/>
  </testcase> <!-- @BeforeClass setUp -->
  <testcase classname="com.proj.base.WebSetUp" name="@AfterClass tearDown" time="-1.673725713344E9">
    <skipped/>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase classname="com.projtest.web.SampleWebTest" name="sampleWebTest" time="0.0">
    <skipped/>
  </testcase> <!-- sampleWebTest -->
  <testcase classname="com.projtest.mobile.demotest.DateTest" name="customdateformate" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.proj.listeners.TestListenerImpl.onTestStart(TestListenerImpl.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- customdateformate -->
  <testcase classname="com.projtest.mobile.demotest.DateTest" name="run" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.proj.listeners.TestListenerImpl.onTestStart(TestListenerImpl.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- run -->
  <testcase classname="com.projtest.mobile.demotest.DateTest" name="sample" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.proj.listeners.TestListenerImpl.onTestStart(TestListenerImpl.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- sample -->
  <testcase classname="com.projtest.mobile.demotest.DateTest" name="splitdate" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.proj.listeners.TestListenerImpl.onTestStart(TestListenerImpl.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- splitdate -->
  <testcase classname="com.projtest.mobile.demotest.SampleTest" name="scrollTest" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.proj.listeners.TestListenerImpl.onTestStart(TestListenerImpl.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- scrollTest -->
  <testcase classname="com.projtest.mobile.demotest.SampleTest" name="test" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.proj.listeners.TestListenerImpl.onTestStart(TestListenerImpl.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase classname="com.projtest.mobile.demotest.LoginPageTest" name="homePageValidation" time="0.0">
    <skipped/>
  </testcase> <!-- homePageValidation -->
  <testcase classname="com.projtest.api.GetUsersTest" name="getUserByPageNumber" time="0.0">
    <skipped/>
  </testcase> <!-- getUserByPageNumber -->
  <testcase classname="com.projtest.api.Post_or_CreateNewUserTest" name="createNewUser" time="0.0">
    <skipped/>
  </testcase> <!-- createNewUser -->
  <testcase classname="com.projtest.api.practice.fileUpload.FileUploadTest" name="uploadPetImage" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.proj.listeners.TestListenerImpl.onTestStart(TestListenerImpl.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- uploadPetImage -->
  <testcase classname="com.projtest.api.practice.fileUpload.FileUploadTest" name="validatePojoToJson" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.proj.listeners.TestListenerImpl.onTestStart(TestListenerImpl.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- validatePojoToJson -->
  <testcase classname="com.projtest.api.practice.dataProvidertest.DataProviderTest" name="dpTest" time="0.015">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.proj.listeners.TestListenerImpl.onTestStart(TestListenerImpl.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- dpTest -->
  <testcase classname="com.projtest.api.practice.dataProvidertest.DataProviderTest" name="testDemo" time="0.002">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.proj.listeners.TestListenerImpl.onTestStart(TestListenerImpl.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testDemo -->
  <testcase classname="com.projtest.api.practice.samplepetstore.BuilderDesignPatternTest" name="builderPattern" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.proj.listeners.TestListenerImpl.onTestStart(TestListenerImpl.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- builderPattern -->
  <testcase classname="com.projtest.api.practice.samplepetstore.BuilderDesignPatternTest" name="builderPattern1" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.proj.listeners.TestListenerImpl.onTestStart(TestListenerImpl.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- builderPattern1 -->
  <testcase classname="com.projtest.api.practice.samplepetstore.BuilderDesignPatternTest" name="builderPattern11" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.proj.listeners.TestListenerImpl.onTestStart(TestListenerImpl.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- builderPattern11 -->
  <testcase classname="com.projtest.api.practice.reporttest.PostResourceTest" name="postResource" time="0.0">
    <skipped/>
  </testcase> <!-- postResource -->
  <testcase classname="com.projtest.api.practice.pojolibtest.serialization.PojoMultipleArrayObjectTest" name="serialization" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.proj.listeners.TestListenerImpl.onTestStart(TestListenerImpl.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- serialization -->
  <testcase classname="com.projtest.api.practice.pojolibtest.serialization.PojoArrayObjectTest" name="serialization" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.proj.listeners.TestListenerImpl.onTestStart(TestListenerImpl.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- serialization -->
  <testcase classname="com.projtest.api.practice.pojolibtest.serialization.PojoLibTest" name="serialization" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.proj.listeners.TestListenerImpl.onTestStart(TestListenerImpl.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- serialization -->
</testsuite> <!-- Command line test -->
